DockerfilePrebuiltBinary

```diff
@@ -1,10 +1,42 @@
-FROM alpine AS spacemesh
+FROM ubuntu:18.04 AS linux
+ENV DEBIAN_FRONTEND noninteractive
+ENV SHELL /bin/bash
+ARG TZ=US/Eastern
+ENV TZ $TZ
+USER root
+RUN bash -c "for i in {1..9}; do mkdir -p /usr/share/man/man\$i; done" \
+ && echo 'APT::Get::Assume-Yes "true";' > /etc/apt/apt.conf.d/90noninteractive \
+ && echo 'DPkg::Options "--force-confnew";' >> /etc/apt/apt.conf.d/90noninteractive \
+ && apt-get update --fix-missing \
+ && apt-get install -qy --no-install-recommends \
+    ca-certificates \
+    tzdata \
+    locales \
+    git \
+    bash \
+    sudo \
+    unzip \
+    make \
+    curl \
+    procps \
+    net-tools \
+    apt-transport-https \
```

Comment:
This makes no sense... why would all this be needed in runtime? We're not gonna require users to install all these dev tools...

Reply:
I removed all staff except porcps (ps command) and net-tools (ping command)